src/extra/MarkdownEditor/defaultStyle.ts

```diff
@@ -207,16 +207,16 @@ export default css`
 	}
 	.CodeMirror-scroll {
 		overflow: scroll !important;
-		margin-bottom: -30px;
```

Comment:
Just to make sure, how this file was changed ?

Reply:
I'm not sure. I don't recall changing it.

Reply:
If this was not intentional then please revert it

Reply:
Please revert the changes to this file then